metrics/grafana/tikv_details.dashboard.py

```diff
@@ -9287,6 +9287,40 @@ def StatusServer() -> RowPanel:
     return layout.row_panel
 
 
+def ResourceControl() -> RowPanel:
+    layout = Layout(title="Resource Control")
+    layout.row(
+        [
+            graph_panel(
+                title="Background Task Total Wait Duration",
+                yaxes=yaxes(left_format=UNITS.MICRO_SECONDS),
+                targets=[
+                    target(
+                        expr=expr_sum_rate(
+                            "tikv_resource_control_background_task_wait_duration",
+                            by_labels=["resource_group"],
```

Comment:
```suggestion
                            by_labels=["instance", "resource_group"],
```
Better by instance as this control is per-instance